std/file.d

```diff
@@ -3925,9 +3936,12 @@ public:
     Params:
         path = The directory to iterate over.
                If empty, the current directory will be iterated.
-        mode = Whether the directory's sub-directories should be iterated
-               over depth-first ($(D_PARAM depth)), breadth-first
-               ($(D_PARAM breadth)), or not at all ($(D_PARAM shallow)).
+
+        mode = Whether the directory's sub-directories should be
+               iterated in depth-first port-order ($(LREF depth)),
+               depth-first pre-order ($(LREF breadth)), or not at all
+               ($(LREF shallow)).
```

Comment:
Then I can't use in with `LREF`.

Maybe there could be a `LREF` variant that links to the new hierarchical anchors, i.e. `HLREF = <a href="#.$1">$(D $1)</a>`, but... more macros...